repos/fdbt-site/package.json

```diff
@@ -17,18 +17,14 @@
         "lint": "eslint . --ext .ts,.tsx",
         "lint:fix": "eslint --fix . --ext .ts,.tsx",
         "lint:cypress": "cd cypress_tests && npm run lint:fix",
-        "postshrinkwrap": "npx --quiet npm-force-resolutions && npm install --no-package-lock --no-audit --no-fund --no-progress --silent && npx prettier --write ./package-lock.json"
+        "postshrinkwrap": "npx --quiet npm-force-resolutions && npm install --no-package-lock --no-audit --no-fund --no-progress --silent && npx prettier --write ./package-lock.json",
+        "precommit": ". ${NVM_DIR}/nvm.sh && nvm use && npm run lint && npm run tsc && npm run test"
```

Comment:
Sorry, could we replace the npm run test with 'CI=true jest --silent'

Reply:
done